code/game/objects/items/devices/explosive_flashbulb.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/flashbulb/bomb
+	name = "suspicious flashbulb"
+	desc = "A powerful flashbulb that looks slightly off, with strange wires running out the back."
+	charges_left = 15
+
+/obj/item/flashbulb/bomb/use_flashbulb()
+	to_chat(usr, "<span class='userdanger'>You press down firmly on the top flash and hear a violent hiss!</span>")
```

Comment:
Grammar mistakes in this sentence. Also, how does this account for non-user activated flashes?

Reply:
It does not, going to remove this since it's not particularly necessary and it's going to be strange with wall flashes